vault/auth_test.go

```diff
@@ -323,7 +405,7 @@ func TestCore_DisableCredential_Cleanup(t *testing.T) {
 		Login:       []string{"login"},
 		BackendType: logical.TypeCredential,
 	}
-	c, _, _ := TestCoreUnsealed(t)
+	c, _, _, _ := TestCoreUnsealedWithMetrics(t)
```

Comment:
You don't have to fix it in this backport, but I will note that all these changes to the test method could be undone now.  :)

Reply:
Absolutely! I can make a separate PR changing those to TestCoreUnsealed.